project/plugins.sbt

```diff
@@ -6,6 +6,8 @@ addSbtPlugin("com.dwijnand"                      % "sbt-dynver"       % "3.3.0")
 addSbtPlugin("com.lightbend.akka"                % "sbt-paradox-akka" % "0.18")
 addSbtPlugin("com.lightbend.paradox"             % "sbt-paradox-project-info" % "1.1.2")
 addSbtPlugin("com.eed3si9n"                      % "sbt-unidoc"       % "0.4.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.0")
+addSbtPlugin("com.lightbend.sbt" % "sbt-publish-rsync" % "0.1")
```

Comment:
Formatting.

Reply:
Remove the space from the others, you mean?

Reply:
Or add spaces to the new ones. Scalafmt can do that.